Entities/Block.cs

```diff
@@ -0,0 +1,21 @@
+using FRITeam.Swapify.Entities.Enums;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace FRITeam.Swapify.Entities
+{
+    public class Block : BaseEntity
+    {
+        public eBlockType BlockType{ get; set; }
+
+        public List<TimeSlot> TimeSlots { get; set; }
+
+        public Course Course { get; set; }
```

Comment:
I think (may be wrong) that entities are transferable and persist structures. If yes, Course is just association, so it should be just its Id here...

Reply:
But I miss Room (string) and teacher(id) also